README.md

```diff
@@ -459,6 +459,7 @@ const observer = {
     }
 
     // videoTileDidUpdate is also invoked when you call startLocalVideoTile or tileState changes.
+    // the tileState.active might be false in poor Internet connection, when the video tile was paused by the user, or when the video tile first arrived.
```

Comment:
How about this, e.g. `The tileState` and no passive voice? (Sorry, I gave you that sentence)
> The tileState.active can be false in poor Internet connection, when the user paused the video tile, or when the video tile first arrived.

Reply:
Updated.